spec/rubocop/cop/rails/find_each_spec.rb

```diff
@@ -67,7 +67,15 @@ class C; User.all.find_each { |u| u.x }; end
     RUBY
   end
 
-  it 'does not register an offense when using order(...) earlier' do
-    expect_no_offenses('User.order(:name).all.each { |u| u.something }')
+  context 'ignored methods' do
+    let(:cop_config) { { 'IgnoredMethods' => %w[order lock] } }
+
+    it 'does not register an offense when using order(...) earlier' do
+      expect_no_offenses('User.order(:name).each { |u| u.something }')
+    end
+
+    it 'does not register an offense when using lock earlier' do
+      expect_no_offenses('User.lock.each { |u| u.something }')
+    end
```

Comment:
Can you add a test that would register an offense when using a method not listed in `IgnoredMethods`?